setup.py

```diff
@@ -4,10 +4,16 @@
 install_requires = [
     'sockjs-tornado==1.0.0',
     'tornado==3.2.2',
-    'wsgiref==0.1.2',
-    'argparse'
+    # 'wsgiref==0.1.2',
+    'argparse',
+    'simplejson' # simplejson is really required for py3 support to avoid encoding problems
 ]
 
+try:
+    import wsgiref
+except ImportError:
+    install_requires.append('wsgiref==0.1.2')
```

Comment:
I believe that we can skip the wsgiref requirement completely. I think that tornado required it before and for some reason I have decided to pin the version in our requirements, but it is not needed anymore.